### PR TITLE
LGB-2 - Google it for me

### PR DESCRIPTION
**Pull request by bengista**
Creation of a function that searches a string after you write certain phrase, for example "who is".

### CODE REVIEW

**Comment by bengista** [comment]
This function doesn't use switch, because it doesn't use PREFIX.

**Review by janfous — APPROVED** [review]

**Review by janfous — CHANGES_REQUESTED** [review]
hold up bitch

File: commands/googlebot/google.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        let reg5 = new RegExp("(\t|kdo je)", "i");
+        let reg6 = new RegExp("(\t|who is)", "i");
+        let reg7 = new RegExp("(\t|who tf is)", "i");
+        let reg8 = new RegExp("(\t|who the fuck is)", "i");
+        let reg9 = new RegExp("(\t|kde je)", "i");
+        let reg10 = new RegExp("(\t|where is)", "i");
+        let reg11 = new RegExp("(\t|where the fuck is)", "i");
+        let reg12 = new RegExp("(what is love|\t)", "i");
+        let reg13 = new RegExp("(who asked)", "i");
+        let reg14 = new RegExp("who am i", "i");
```
Comment (janfous) [inline commands/googlebot/google.js]: what the fuck

File: commands/googlebot/google.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("Vygooglil jsem to za tebe:", embedLink, true);
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("I googled it for you:", embedLink, true)
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("Vygooglil jsem to za tebe:", embedLink, true)
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("I googled it for you:", embedLink, true)
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("I googled it for you:", embedLink, true)
+
+            message.channel.send(messageBlock);        }
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("Vygooglil jsem to za tebe:", embedLink, true)
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+            let subject = string.replace(/\s+/g, "+");
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("I googled it for you:", embedLink, true)
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

File: commands/googlebot/google.js [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+
+
+            const vars = ['https://www.google.com/search?q=', subject.trim()];
+            const embedLink = vars.join('');
+
+            const messageBlock = new Discord.MessageEmbed()
+                .setColor("BLUE")
+                .addField("I googled it for you:", embedLink, true)
+
+            message.channel.send(messageBlock);
```
Comment (janfous) [inline commands/googlebot/google.js]: copy/paste

**Review by janfous — CHANGES_REQUESTED** [review]

File: commands/googlebot/google.js (hunk 1 above)
Comment (bengista) [reply commands/googlebot/google.js]: Your mom

**Review by janfous — APPROVED** [review]

**Pull request merged**